simulation/simulation.py

```diff
@@ -66,4 +83,6 @@
 
 SIMULATION.run(duration=timedelta(hours=ARGS.duration), time_step=600)
 
-SIMULATION.plot(fast=True, legend=True, filename=os.path.join("/code", "leeway", ARGS.id))
+OUTFILE = os.path.join("/code", "leeway", ARGS.id)
```

Comment:
I think we should add a dedicated output directory: `os.path.join("/code", "leeway", "output", ARGS.id)`